### PR TITLE
bumped hyper to 0.10 and cookie to 0.6, bumped version to 0.19.1

### PR DESCRIPTION
**Pull request by avadacatavra**
happy to hear suggestions about improving the into function

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/webdriver-rust/70)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 description = "Library implementing the wire protocol for the W3C WebDriver specification."
 repository = "https://github.com/jgraham/webdriver-rust"
 license = "MPL-2.0"
 
 [dependencies]
 backtrace = "0.3"
-cookie = {version = "0.2", default-features = false}
-hyper = {version = "0.9", default-features = false}
+cookie = {version = "0.6", default-features = false}
+hyper = "0.10"
```
Comment (andreastt) [inline Cargo.toml]: This presumably enables SSL to be compiled into hyper, which will fail when we cross compile [geckodriver](https://github.com/mozilla/geckodriver) to Windows x86 and x86_64.

**Comment by jgraham** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[Cargo.toml, line 4 at r1](https://reviewable.io:443/reviews/mozilla/webdriver-rust/70#-Kd_v0MZ5r0qmZIzaPYI:-Kd_v0M_qfhmUprgUmQA:b-geo1wq) ([raw file](https://github.com/mozilla/webdriver-rust/blob/ec76c17775048237f7502f52c72a655eefbbdef0/Cargo.toml#L4)):*
> ```TOML
> 
> name = "webdriver"
> version = "0.19.1"
> ```

Just leave the version number at the moment and we can change it in another commit when we release.

---

*[Cargo.toml, line 13 at r1](https://reviewable.io:443/reviews/mozilla/webdriver-rust/70#-KdXzDgJByPpTWp-Qr0L-r1-13:-KdaIlTqlaflnkhC76QA:b-grjkg7) ([raw file](https://github.com/mozilla/webdriver-rust/blob/ec76c17775048237f7502f52c72a655eefbbdef0/Cargo.toml#L13)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

This presumably enables SSL to be compiled into hyper, which will fail when we cross compile [geckodriver](https://github.com/mozilla/geckodriver) to Windows x86 and x86_64.
</blockquote></details>

No, the SSL dependency was removed in 0.10 per https://github.com/hyperium/hyper/blob/master/CHANGELOG.md

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/webdriver-rust/70)*

File: Cargo.toml (hunk 1 above)
Comment (andreastt) [inline Cargo.toml]: > No, the SSL dependency was removed in 0.10 per https://github.com/hyperium/hyper/blob/master/CHANGELOG.md

Excellent!

**Comment by avadacatavra** [comment]
@jgraham i reverted the webdriver version bump

**Comment by jgraham** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/webdriver-rust/70)*

**Pull request merged**